src/scripts/actions.js

```diff
@@ -1,11 +1,11 @@
-import {SQUARE_SIZE, TWEEN_DURATION} from './const';
+import {SQUARE_SIZE, TWEEN_DURATION, S} from './const';
 import {tween, tweenRotation, interp} from './tween';
 
-export function actionLine(room, action, from, to) {
+export function actionLine(room, action, from, to, color) {
 	let g = new PIXI.Graphics();
 	room.g.addChild(g);
 
-	const color = 0xffff00;
+	color = color || 0xffff00;
```

Comment:
This should be added as a default value to the color parameter above, like this: `, color = 0xffff00) {`